akka-actor/src/main/java/akka/annotation/ApiMayChange.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.annotation;
+
+import java.lang.annotation.*;
+
+/**
+ * Marks APIs that are meant to evolve towards becoming stable APIs, but are not stable APIs yet.
+ * 
+ * Evolving interfaces can change from one feature release to another release (i.e. 2.1 to 2.2) without up-front notice.
+ * They may also be removed with or without replacement.
+ * 
+ * In general APIs marked like this are released as part of stable modules
+ */
+@Documented
+@Retention(RetentionPolicy.CLASS) // to be accessible by MiMa
+@Target({ElementType.METHOD, ElementType.CONSTRUCTOR, ElementType.FIELD, ElementType.TYPE, ElementType.PACKAGE})
+public @interface ApiMayChange {
```

Comment:
Should we also encourage to include some kind of Disclaimer as first line in the javadoc/scaladoc?